medium-tests/src/test/java/mediumtest/ConnectedHotspotMediumTests.java

```diff
@@ -66,8 +67,8 @@ void foo() {
       .withPlugin(TestPlugin.JAVA)
       .start();
     var backend = harness.newBackend()
-      .withFullSynchronization()
-      .withSecurityHotspotsEnabled()
+      .withBackendCapability(BackendCapability.FULL_SYNCHRONIZATION)
+      .withBackendCapability(BackendCapability.SECURITY_HOTSPOTS)
```

Comment:
Nitpick: this method could accept a var arg. This would make tests a little less verbose